kolibri/plugins/epub_viewer/assets/src/views/SettingsSideBar.vue

```diff
@@ -323,20 +348,25 @@
         context:
           "The EPUB reader allows learners to set the background of the reader to different shades of colors using the 'Theme' option. In this case it can be set to black.",
       },
-      setCustomTheme: {
-        message: "Set custom theme '{themeName}'",
+      setYellowTheme: {
+        message: 'Set yellow theme',
         context:
-          "The EPUB reader allows learners to set the background of the reader to different shades of user preferred colors using the 'My themes' option. In this case it can be set to {themeName}.",
+          "The EPUB reader allows learners to set the background of the reader to different shades of colors using the 'Theme' option. In this case it can be set to yellow.",
       },
-      deleteCustomTheme: {
-        message: "Delete custom theme '{themeName}'",
+      setBlueTheme: {
+        message: 'Set blue theme',
         context:
-          "The EPUB reader allows learners to set the background of the reader to different shades of user preferred colors using the 'My themes' option. In this case it can be deleted.",
+          "The EPUB reader allows learners to set the background of the reader to different shades of colors using the 'Theme' option. In this case it can be set to blue.",
       },
-      editCustomTheme: {
-        message: "Edit custom theme '{themeName}'",
+      closeSideBar: {
+        message: 'Close settings',
         context:
-          "The EPUB reader allows learners to set the background of the reader to different shades of user preferred colors using the 'My themes' option. In this case it can be edited.",
+          'Used to close the settings button where a learner can adjust things like the text size or the background color.',
+      },
+      sideBarTitle: {
+        message: 'THEMES & SETTINGS',
```

Comment:
Let's have this as "Themes & settings" and use styling to make it all caps in the UI, if we need to. Generally we don't use all caps, but if it's in the spec, we can for now. But, we are moving away from this pattern (i.e. with our buttons, etc.) so it would be good to use sentence case and have the flexibility in the future

Reply:
Good 👁️ @marcellamaki , I missed this completely! Yes, please, we need to steer away from all caps!

Reply:
I'm also now thinking we don't really need the new string 'Themes & settings', let's just keep it at 'Settings'. 

And the H3 should be 'Theme**s**', in plural:

![2023-09-06_19-57-35](https://github.com/learningequality/kolibri/assets/1457929/7ec1c2ff-f24f-45de-bd56-4fbbccffeadf)

Reply:
- 'THEMES & SETTINGS' is changed to 'Settings'
- 'Theme' is changed to 'Default themes'

Reply:
I'm sorry, probably should have mentioned that: even though we have a few instances of the word 'Default' in Kolibri UI, we decided to restrict them to the admin pages, and not use them in the learner view to keep the language and terminology as simple and readable as possible. Just 'Themes' should be OK.